mbtools/inject_ib_ids.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+from pathlib import Path
+from mbtools.models import MoodleHtmlElement
+from lxml import etree
+
+CLASSES_TO_INJECT = [
+    "os-raise-ib-input",
+    "os-raise-ib-pset",
+    "os-raise-ib-pset-problem"]
+
+
+def inject_ib_ids(html_dir):
+    all_files = []
+    for path in Path(html_dir).rglob('*.html'):
+        all_files.append(path)
```

Comment:
Minor: In general it's not a great idea to queue things up like this vs just iterate as memory usage will be `O(n)` vs `O(1)`. The following structure would avoid that and be more succinct:

```python
def inject_ib_ids(html_dir):
    for path in Path(html_dir).rglob('*.html'):
        with open(path, 'r') as f:
```